### ISSUE TITLE
Firefox: Balloon behaves oddly if dragged with mouse after being dragged with keyboard nav

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/balloons-and-static-electricity#553]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Firefox (Not the same on Chrome)
**Problem description**
For https://github.com/phetsims/qa/issues/721
If you move the balloon around with keyboard nav (and give it charge) and then drop it, then pick it up with mouse, the balloon acts oddly. It seems to think it has been dropped, picked up, and dropped again continually. Dropping it and picking it up again seems to resolve this.

**Steps to reproduce**
1. Use keyboard to pick up the balloon and give it a charge
2. Drop the balloon with keyboard nav (bug will happen even without this step)
3. Pick up the balloon with mouse interaction and drag the balloon around

**Visuals**
![ezgif com-gif-maker](https://user-images.githubusercontent.com/41024075/138513772-2acaff16-f2c0-416b-8c59-36c62c4d2ba1.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.2/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-rc.2 2021-10-11 21:55:48 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0
Language: en-US
Window: 1280x667
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (ANGLE (Intel(R) HD Graphics Direct3D11 vs_5_0 ps_5_0))
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks for investigating this @zepumph.

Yes, the PressListener is the only way to access the GrabDragInteraction with mouse/touch. I can't think of anything related to multitouch that would be problematic here. I tried to think through how this might work with iOS VoiceOver but I couldn't think of any problems with this. I tested on iOS 15 with VoiceOver and verified that GrabDragInteraction still worked.

This was an improvement but there was another case this is still an issue, regardless of platform:
1) Charge the balloon (because it is easier to see the problem)
2) Pick up the charged balloon with keyboard.
3) (Without clicking anywhere else in the sim) click on the balloon with a mouse to pick it up, leave mouse pressed.
4) Press spacebar while the mouse is pressed. This will turn the balloon back into a grabbable, but the balloon's DragListener is still pressed for dragging.

I thought about moving the `!this.pressListener.isPressedProperty.value` to `releaseDraggable`. But we need to be able to release while actively dragging sometimes. Ideally, we would be able to interrupt the DragListener in this case but GrabDragInteraction doesn't have knowledge of the DragListener. Maybe that needs to change.

**Comment by jbphet** [comment]
Over Slack, @jessegreenberg also said:

> If it was decided that publishing with this bug is OK, I would probably recommend proceeding without this fix. I think we should work on it a bit more before putting the change in RC SHAs.

Based on @jessegreenberg's review and recommendation, I'm going to move forward with publishing the next RC off of the 1.5 branch without this fix.  If there is a breakthrough before publication, we can consider working this in.

**Comment by jessegreenberg** [comment]
I made https://github.com/phetsims/scenery-phet/issues/710 to work on this generally since it isn't specific to BASE (though BASE probably has the most obvious case since the balloon can move on its own).

**Comment by jbphet** [comment]
@jessegreenberg - Since it sounds like this issue will be addressed in common code, and we've decided not to have the problem block the 1.5 release of BASE, can this issue just be closed, or at least reduced in priority?

**Comment by jessegreenberg** [comment]
Indeed, reducing priority since this was published with this behavior.

**Comment by KatieWoe** [comment]
I'm not seeing the behavior on main as of 6/4/26

**Comment by jessegreenberg** [comment]
Great, thank you @KatieWoe ! Closing.
